documentation/API.md

```diff
@@ -224,13 +226,15 @@ Returns (on success):
             "name": "applyTestRego",
             "display_name": "Apply Test Rego",
             "description": "Permission for external user to apply for a Test template of user registration",
-            "is_user_granted": true
+            "is_user_granted": true,
+            "template_codes": [ "UserRegistration" ]
```

Comment:
Can you just update the docs to match the camelCase fields now?

Reply:
Oh! my bad... this should have been fixed.